extensions/map-layers-auth/src/ArcGis/ArcGisAccessClient.ts

```diff
@@ -0,0 +1,365 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Tiles
+ */
+
+import { assert, BeEvent } from "@itwin/core-bentley";
+import { MapLayerAccessClient, MapLayerAccessToken, MapLayerAccessTokenParams, MapLayerTokenEndpoint } from "@itwin/core-frontend";
+import { ArcGisOAuth2Token, ArcGisTokenClientType } from "./ArcGisTokenGenerator";
+import { ArcGisOAuth2Endpoint, ArcGisOAuth2EndpointType } from "./ArcGisOAuth2Endpoint";
+import { ArcGisTokenManager } from "./ArcGisTokenManager";
+import { ArcGisUrl } from "./ArcGisUrl";
+
+/** @alpha */
```

Comment:
We want people to try these new features, right? If so please replace `alpha` with `beta`.

Reply:
[Release tag guidelines](https://github.com/iTwin/itwinjs-core/blob/master/docs/learning/guidelines/release-tags-guidelines.md).
IMO `alpha` is the least useful. It means the feature is so experimental we don't want users to know about nor use it.
Most new features should go straight to `beta`. Beta APIs are included in documentation. We encourage users to try them out and provide feedback. User feedback is essential for validating the implementation and stabilizing the API. API stability is only guaranteed for `public` APIs.